packages/tools/replay-tool/src/replayMessages.ts

```diff
@@ -797,25 +837,32 @@ export class ReplayTool {
         }
     }
 
-    private compareSnapshots(contentAsString: string, filename: string) {
+    private compareWithReferenceSnapshot(snapshot: IFileSnapshot, referenceSnapshotFilename: string) {
+        // Read the reference snapshot and covert it to normalized IFileSnapshot.
+        const referenceSnapshotString = fs.readFileSync(`${referenceSnapshotFilename}.json`, "utf-8");
+        const referenceSnapshot = getNormalizedFileSnapshot(JSON.parse(referenceSnapshotString));
```

Comment:
won't this sort everything? Things like the segments of a merge tree shouldn't be sorted, order is important

Reply:
The tool only sorts specific blob contents. Currently, it only sorts garbage collection blobs with path ".gc".
As we discussed offline, adding a config to the API that accepts paths of blobs to be sorted will make this more clear and extensible. I will work on that.